modules/phase_field/src/kernels/ACBulk.C

```diff
@@ -10,44 +10,59 @@ template<>
 InputParameters validParams<ACBulk>()
 {
   InputParameters params = validParams<KernelValue>();
-  params.addClassDescription("Allen-Cahn Kernel");
+  params.addClassDescription("Allen-Cahn bulk energy kernel");
   params.addParam<MaterialPropertyName>("mob_name", "L", "The mobility used with the kernel");
+  params.addCoupledVar("args", "Vector of additional arguments");
```

Comment:
I don't think this doc string is sufficiently clear. But we should merge this for now and go over the pf code. I suggest adding something along the lines of
```
List of non-linear variables this object depends on
```